SPECS/qemu/qemu.spec

```diff
@@ -1664,9 +1664,13 @@ rm -rf %{buildroot}%{_datadir}/%{name}/openbios-sparc32
 rm -rf %{buildroot}%{_datadir}/%{name}/openbios-sparc64
 # Provided by package SLOF
 rm -rf %{buildroot}%{_datadir}/%{name}/slof.bin
+
+%ifarch aarch64
```

Comment:
Please remove this block.  ipxe only builds for x86_64.  And we really should be consistent here anyway.
qemu.